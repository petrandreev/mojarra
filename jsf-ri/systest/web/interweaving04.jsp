<!--
 Copyright 2004 Sun Microsystems, Inc. All rights reserved.
 SUN PROPRIETARY/CONFIDENTIAL. Use is subject to license terms.
-->

<%@ page contentType="text/html" language="java" %>
<%@ taglib prefix="c" uri="http://java.sun.com/jsp/jstl/core" %>
<%@ taglib prefix="f" uri="http://java.sun.com/jsf/core" %>
<%@ taglib prefix="h" uri="http://java.sun.com/jsf/html" %>

<%
    pageContext.setAttribute("cond", Boolean.TRUE,
                             PageContext.REQUEST_SCOPE);
 
%>

<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN">
<f:view>
<html>
<head>
<title>jstl-if-without-id</title>
</head>
<body>
<h:outputText value="[First]"/>
<c:if test="${requestScope.cond}">
  <h:outputText value="[Second]"/>
</c:if>
<h:outputText value="[Third]"/>
</body>
</html>
</f:view>
