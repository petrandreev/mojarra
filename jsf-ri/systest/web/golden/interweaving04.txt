<!--
 Copyright 2004 Sun Microsystems, Inc. All rights reserved.
 SUN PROPRIETARY/CONFIDENTIAL. Use is subject to license terms.
-->








<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN">

<html>
<head>
<title>jstl-if-without-id</title>
</head>
<body>
[First][Second][Third]

  


</body>
</html>

